src/app/(core)/(home)/commissions.tsx

```
@@ -73,17 +81,15 @@ export function InfiniteCommissions() {
                             <CommissionCard commission={commission} />
                         </motion.div>
                     ))
-                )}
+                })}
             </motion.div>
             <div ref={ref}></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)